benefit/data/sheets/FixedLists/FixedLists.json

```diff
@@ -181,6 +181,8 @@
   {"LiveFrom": "01/01/2018", "ID": "FL_dwpStates", "ListElementCode": "Registered", "ListElement": "Appeal registered (DWP)"},
   {"LiveFrom": "01/01/2018", "ID": "FL_dwpStates", "ListElementCode": "appealAbated", "ListElement": "Appeal abated"},
   {"LiveFrom": "01/01/2018", "ID": "FL_dwpStates", "ListElementCode": "adjournmentNoticeIssued", "ListElement": "Adjournment notice issued"},
+  {"LiveFrom": "01/01/2018", "ID": "FL_dwpStates", "ListElementCode": "hearingRecordingProcessed", "ListElement": "Hearing Recording Processed"},
```

Comment:
Can we change the list elements to be lower case so they are consistent with all others?